src/renderer.js

```diff
@@ -6,41 +7,54 @@ function getMapping() {
     ]
 }
 function init() {
-    window.config = {};
-    window.config = window.electron.getConfig();
     addListeners();
     populateFields();
 }
 
 async function populateFields() {
-    let config = window.config;
-    let mapping = getMapping();
-    for (map of mapping) {
-        let input = document.getElementById(map.id);
-        input.setAttribute("value", config[map.key])
-    }
+    config = window.electron.getConfig();
+    let profile = document.getElementById("profile");
+    profile.innerHTML = "";
+
+    let options = Object.keys(config).map(function(name) {
+        return `<option value="${name}">${name}</option>`
+    });
+    profile.innerHTML += options.join('');
+
+}
+
+function getActiveConfig() {
+    let profile = document.getElementById("profile");
+    var name = profile.options[profile.selectedIndex].value;
+    console.log('name', name);
```

Comment:
could you check similar console.log, and see if they are really necessary to remove. If it's for dev purpose, you can add some TODO comments, so people who work on this later will know and remove.